testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java

```diff
@@ -221,6 +256,66 @@ private void safeCloseClient() {
     }
 
     /**
+     * Reload current server
+     * This method makes sure client on server is still operational after reload is done.
+     * @param adminOnly tells server to boot in admin only mode or normal
+     * @param timeout time in miliseconds to wait for server to come back after reload
+     */
+    public void reload(boolean adminOnly,int timeout){
+        executeReload(adminOnly);
+        waitForLiveServerToReload(timeout); //30 seconds
```

Comment:
Use org.jboss.as.test.integration.management.util.ServerReload, which has been ported from full to core.